cvat/apps/auto_annotation/model_manager.py

```diff
@@ -144,16 +148,26 @@ def save_file_as_tmp(data):
         dl_model_id = create_empty(owner=owner)
 
     run_tests = bool(model_file or weights_file or labelmap_file or interpretation_file)
+    if is_custom in ["tensorflow","maskrcnn"]:
```

Comment:
I see that these string constants are used many times, could you add enum for them?

Reply:
Sure, do we have any specific file where we should create enum class? or do you have any suggestion where this should be created?

Reply:
I think this file will be ok.

Reply:
ok thanks